packages/cli-repl/package.json

```diff
@@ -95,9 +96,14 @@
     "@types/text-table": "^0.2.1",
     "@types/yargs-parser": "^15.0.0",
     "chai-as-promised": "^7.1.1",
+    "depcheck": "^1.4.3",
     "lodash": "^4.17.21",
     "moment": "^2.29.1",
-    "mongodb-crypt-library-dummy": "^1.0.2"
+    "mongodb": "^5.6.0",
+    "mongodb-crypt-library-dummy": "^1.0.2",
+    "node-fetch": "^2.6.1",
+    "rimraf": "^5.0.1",
```

Comment:
Opened https://jira.mongodb.org/browse/MONGOSH-1500 to get rid of `rimraf` where we can use `fs.rm` instead